### ISSUE TITLE
More accessibility errors

### ISSUE THREAD

**Issue by mgifford** [squidfunk/mkdocs-material#6838]
### Context

Basic documentation examples should meet WCAG 2.2 AA (but at least WCAG 2.0 at this point). 

Here are some errors from:

https://squidfunk.github.io/mkdocs-material/getting-started/

### Bug description

Title: WCAG 1.4.3: Ensures the contrast between foreground and background colors meets WCAG 2 AA minimum contrast ratio thresholds (.md-banner__inner)
Tags: Accessibility, WCAG 1.4.3, color-contrast

Issue: Ensures the contrast between foreground and background colors meets WCAG 2 AA minimum contrast ratio thresholds (color-contrast - https://accessibilityinsights.io/info-examples/web/color-contrast)

Target application: Installation - Material for MkDocs - https://squidfunk.github.io/mkdocs-material/getting-started/

Element path: .md-banner__inner

Snippet: <div class="md-banner__inner md-grid md-typeset">

Related paths: 
  .md-banner
  body

How to fix: 
Fix any of the following:
  Element has insufficient color contrast of 4.49 (foreground color: #7f8184, background color: #17191f, font size: 10.5pt (14px), font weight: normal). Expected contrast ratio of 4.5:1

Environment: Microsoft Edge version 122.0.0.0

====

This accessibility issue was found using Accessibility Insights for Web 2.41.0 (axe-core 4.7.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.

Title: WCAG 1.4.1: Ensure links are distinguished from surrounding text in a way that does not rely on color (a[href$="www.mkdocs.org"])
Tags: Accessibility, WCAG 1.4.1, link-in-text-block

Issue: Ensure links are distinguished from surrounding text in a way that does not rely on color (link-in-text-block - https://dequeuniversity.com/rules/axe/4.7/link-in-text-block?application=msftAI)

Target application: Installation - Material for MkDocs - https://squidfunk.github.io/mkdocs-material/getting-started/

Element path: p:nth-child(4) > a[href$="www.mkdocs.org"]

Snippet: <a href="https://www.mkdocs.org">MkDocs</a>

Related paths: 
  article > p:nth-child(4)

How to fix: 
Fix any of the following:
  The link has insufficient color contrast of 1.87:1 with the surrounding text. (Minimum contrast is 3:1, link text: #5e8bde, surrounding text: #bfc1c6)
  The link has no styling (such as underline) to distinguish it from the surrounding text

Environment: Microsoft Edge version 122.0.0.0

====

This accessibility issue was found using Accessibility Insights for Web 2.41.0 (axe-core 4.7.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.

Title: WCAG 2.1.1: Ensure elements that have scrollable content are accessible by keyboard (.md-search__scrollwrap)
Tags: Accessibility, WCAG 2.1.1, scrollable-region-focusable

Issue: Ensure elements that have scrollable content are accessible by keyboard (scrollable-region-focusable - https://accessibilityinsights.io/info-examples/web/scrollable-region-focusable)

Target application: Installation - Material for MkDocs - https://squidfunk.github.io/mkdocs-material/getting-started/

Element path: .md-search__scrollwrap

Snippet: <div class="md-search__scrollwrap"> <div class="md-search-result" data-md-component="search-result"> <div class="md-search-result__meta">Type to start searching</div> <ol class="md-search-result__list" role="presentation"></ol> </div> </div>

How to fix: 
Fix any of the following:
  Element should have focusable content
  Element should be focusable

Environment: Microsoft Edge version 122.0.0.0

====

This accessibility issue was found using Accessibility Insights for Web 2.41.0 (axe-core 4.7.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.

### Related links

- https://squidfunk.github.io/mkdocs-material/getting-started/
- https://accessibilityinsights.io/

### Reproduction

Just run this extension https://accessibilityinsights.io on your example pages. 

### Steps to reproduce

1) https://squidfunk.github.io/mkdocs-material/getting-started/
2) Run https://accessibilityinsights.io

### Browser

Edge

### Before submitting

- [X] I have read and followed the [bug reporting guidelines](https://squidfunk.github.io/mkdocs-material/contributing/reporting-a-bug/).
- [X] I have attached links to [the documentation](https://squidfunk.github.io/mkdocs-material/), and possibly related [issues](https://github.com/squidfunk/mkdocs-material/issues) and [discussions](https://github.com/squidfunk/mkdocs-material/discussions).
- [X] I assure that I have [removed all customizations](https://squidfunk.github.io/mkdocs-material/contributing/reporting-a-bug/#remove-customizations) before submitting this bug report.
- [X] I have attached a __.zip file__ with a [minimal reproduction](https://squidfunk.github.io/mkdocs-material/guides/creating-a-reproduction/).

**Comment by squidfunk** [comment]
Thanks for reporting. This issue contains essentially the same problems as #6837, with the addition of the announcement bar (`.md-banner`). Please understand that those are __custom styles__, i.e., [they come from a CSS](https://squidfunk.github.io/mkdocs-material/assets/stylesheets/custom.14cc6f30.min.css) file that we __only use on our own documentation__. For this reason, it is not an issue with the project itself. We'll look into if we can make it any better, but it's not a problem with the styles as provided by Material for MkDocs. Closing as non-actionable.
